pysand/erosion.py

```diff
@@ -1,7 +1,32 @@
 import numpy as np
+import pandas as pd
+import logging
+import pysand.exceptions as exc
 
+logger = logging.getLogger(__name__)
 # Models from DNVGL RP-O501, equation references in parenthesis
 
+
+def validate_inputs(v_m, Q_s, rho_m=30, mu_m=1.5e-5, GF=3, D=0.5, d_p=1):
```

Comment:
You should probably do: 
`def validate_inputs(arguments):`
then later in function:
`if 'd_p' in arguments:` and then validate d_p. Access d_p value with `arguments['d_p']`
This way the validation function can be general and you can have all validation here!

Reply:
I got all your comments (will make a proper change in a code) except this one. I don't really understand how it is going to work, By arguments do you mean *args (list) or **kwargs (dictionary)?

Reply:
I was thinking about using a dictionary. The idea is to not specify the inputs specifically.. i thought you could use your existing dict as a single object, but maybe you need **kwargs... not sure!

Reply:
Yes, I am remaking it with **kwargs (dictionary), at least trying to :-)